src/mongoose_async_pools.erl

```diff
@@ -44,6 +50,27 @@ config_spec() ->
 pool_name(HostType, PoolId) ->
     persistent_term:get({?MODULE, HostType, PoolId}).
 
+-spec sync(mongooseim:host_type(), pool_id()) -> [pid()].
+sync(HostType, PoolId) ->
+    Pids = mongoose_async_pools:get_workers(HostType, PoolId),
+    rpc:pmap({?MODULE, call_sync}, [HostType], Pids).
+
+call_sync(Pid, HostType) ->
+    try
+        gen_server:call(Pid, sync)
+    catch C:E:S ->
```

Comment:
it is executed inside rpc:pmap, which has try..cactch in it.
Which catches and **just returns badrpc**, which is _bad_.
So, we have to have try catch and logging.

Reply:
Then why not try `rpc:parallel_eval/1`or so? 🤔

Reply:
because it runs it from several nodes, wtf

```
parallel_eval(FuncCalls) -> ResL
Types
FuncCalls = [{Module, Function, Args}]
Module = module()
Function = atom()
Args = ResL = [term()]

Evaluates, for every tuple in FuncCalls, apply(Module, Function, Args) on some node in the network. Returns the list of return values, in the same order as in FuncCalls.
```

btw, pmap uses parallel_eval, so we need a separate pmap